Palashio/libra
# NLP tests

---
name: Added tests to NLP
about: Select this if you're adding a new feature to libra.
labels: feature
---



This pull request closes #290  .

**- What I did**
Added tests to NLP

**- How I did it**
Matched tests of other queries
**- How to verify it**

Tested all tests to make sure they work as intended

- [ ] I updated the docs.

This pull request adds a new feature to libra. @Palashio, could you please take a look at it?
